src/bits.hpp

```diff
@@ -261,12 +261,17 @@ class BitsGeneric {
 
     BitsGeneric<T> Rotl(uint32_t shift) const
```

Comment:
it turns out this function doesn't seem to be used anywhere, so I imagine it would be better to remove it than to add unit tests to it

Reply:
I agree with removing it.